cli/cmd/lql.go

```diff
@@ -19,71 +19,217 @@
 package cmd
 
 import (
+	"fmt"
+	"io"
 	"io/ioutil"
+	"net/http"
+	"strings"
 
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/api"
 )
 
 var (
-	lqlFile string
+	lqlEnd      string
+	lqlEnv      bool
+	lqlFile     string
+	lqlRepo     bool
+	lqlStart    string
+	lqlURL      string
+	lqlValidate bool
 
-	// lqlCmd represents the lql command
+	// lqlCmd represents the lql parent command
 	lqlCmd = &cobra.Command{
-		Use:    "lql <query>",
-		Hidden: true,
-		Short:  "run an LQL query",
+		Aliases: []string{"lql"},
+		Use:     "query",
+		Short:   "Run and manage LQL queries",
+		Long: `Run and manage LQL queries.
+
+NOTE: This feature is not yet available!`,
+	}
+
+	// lqlRunCmd represents the lql run command
+	lqlRunCmd = &cobra.Command{
+		Use:   "run <query|queryID>",
```

Comment:
We use `<param>` for a required parameter and `[param]` for an optional parameter.

We also use `query_id` underscores.

Example:

- https://github.com/lacework/go-sdk/blob/main/cli/cmd/vuln_container.go#L40
- https://github.com/lacework/go-sdk/blob/main/cli/cmd/vuln_container.go#L63

This should look like:
```suggestion
		Use:   "run [query|query_id]",
```

Reply:
Are you considering this optional because we will open the editor?

Reply:
I think optional makes sense here.  I'll update everything as needed.

Reply:
Fixed.